SMCKit/SMC.swift

```diff
@@ -1167,6 +1370,46 @@ public struct SMC {
     */
     private func callSMC(inout inputStruct : SMCParamStruct,
                          inout outputStruct: SMCParamStruct) -> kern_return_t {
+        return callSMC(&inputStruct, outputStruct: &outputStruct, selector: Selector.kSMCHandleYPCEvent.rawValue)
+    }
+    
+    /**
+    Make a call to the SMC
+    
+    - parameter inputStruct: Struct that holds data telling the SMC what you want
+    - parameter outputStruct: Struct holding the SMC's response
+    - returns: IOKit return code
+    */
+    private func callSMC(inout inputStruct : SMCParamStruct,
+        inout outputStruct: SMCParamStruct, selector: UInt32) -> kern_return_t {
```

Comment:
Thanks ! BTW, is is only the second merge with conflicts I make, and my first project with Swift.